src/trio/socket.py

```diff
@@ -1,14 +1,13 @@
+from __future__ import annotations
+
 # This is a public namespace, so we don't want to expose any non-underscored
 # attributes that aren't actually part of our public API. But it's very
 # annoying to carefully always use underscored names for module-level
 # temporaries, imports, etc. when implementing the module. So we put the
 # implementation in an underscored module, and then re-export the public parts
 # here.
 # We still have some underscore names though but only a few.
-
-
 # Uses `from x import y as y` for compatibility with `pyright --verifytypes` (#2625)
-
 # Dynamically re-export whatever constants this particular Python happens to
```

Comment:
Is there a reason for removing whitespace? The comments are distinct so I think it was better with that (or maybe replacing the empty lines with `#`?)

Reply:
No, I can fix that.

Reply:
Should be fixed with [46b0c34](https://github.com/python-trio/trio/pull/2932/commits/46b0c343e675e50d22a2141c3997b655d3021c61)